### DISCUSSION TITLE
Disable and enable UWP apps systemwide

### DISCUSSION THREAD

**Discussion by Anixx** [valinet/ExplorerPatcher#985] [Ideas]
I suggest the following option: possibility to disable UWP apps so that they not to take resources.

This registry key does this:

[HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Image File Execution Options\ApplicationFrameHost.exe]
"Debugger"="c:\\Windows\\System32\\rundll32.exe"

this reverts back:

[-HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Image File Execution Options\ApplicationFrameHost.exe]
"Debugger"="c:\\Windows\\System32\\rundll32.exe"

**Comment by sdw4938** [comment]
Not a good idea as most stuff breaks if UWP apps are disabled.
You wouldn't have a start menu, Taskbar (If you are on Win11), action center, Task View, internet flyout, battery flyout, wifi flyout, clock flyout, oobe, settings, and other first and third party UWP apps.

**Reply by OrthodoxWindows** [reply]
I don't understand, because most of the elements (action centery, flyouts, start menu) you mention do not depend on ApplicationFrameHost.exe, are therefore not concerned by this. The only problem is the settings app (it would be necessary to recreate .cpl type property pages for the management of the settings which did not exist before Windows 10).

Moreover, it is possible to replace all the flyouts by their Windows 7 equivalents, except network flyout, where to use PeNetworkManager. https://www.penetworkmanager.de/
